Sources/DbPackages-postgresql.php

```diff
@@ -480,7 +480,7 @@ function smf_db_change_column($table_name, $old_column, $column_info)
 		else
 			$default = '\'' . $smcFunc['db_escape_string']($column_info['default']) . '\'';
 
-		$action = $column_info['default'] !== null ? 'SET DEFAULT ' . $default : 'DROP DEFAULT';
+		$action = $default !== '' ? 'SET DEFAULT ' . $default : 'DROP DEFAULT';
```

Comment:
this goesa now in direction of nit-picking so feel free to ignore,
but i would like to see that this ``$default === ''`` operator is the like in pg,
so choose what you want === or !== but use this one in both files.